docs/content/latest/admin/yb-admin.md

```diff
@@ -1008,8 +1008,8 @@ yb-admin \
 ```
 
 * _master-addresses_: Comma-separated list of YB-Master hosts and ports. Default value is `localhost:7100`.
-* *placement_info*: Comma-delimited list of placements for *cloud*.*region*.*zone*. Default value is `cloud1.datacenter1.rack1`.
-* *replication_factor*: The number of replicas for each tablet.
+* *placement_info*: Comma-delimited list of placements for *cloud*.*region*.*zone*. Optionally, a minimum replica count can be specified after a colon, the default being 1. It is not recommended to repeat placement multiple times but instead specify the total count after the colon. However, in the event that the user specifies a placement multiple times, the total count from all mentions is taken.
+* *replication_factor*: The number of replicas for each tablet. This value should be more than the total of replica counts specified in *placement_info*.
```

Comment:
Nit: Shouldn't the line be: `This value should be greater equal to the sum total of replica counts...`